security-advisories/mbedtls-security-advisory-2024-01-2.md

```diff
@@ -0,0 +1,31 @@
+# Buffer overflow in mbedtls_x509_set_extension()
+
+**Title** |  Buffer overflow in mbedtls_x509_set_extension().
+---|---
+**CVE** |  CVE-2024-23775
+**Date** |  9th of January, 2024
```

Comment:
Sigh, we don't seem to have a consistent date format. Last year we used `05 October 2023`, in 2022 we used `2022-07-11`, in 2021 `14th of December, 2021`, so this is at least consistent with 2021

Reply:
I will admit to copying a 2021 one, but I thought that *was* the standard...